### ISSUE TITLE
Undefined variable in openshift_logging role

### ISSUE THREAD

**Issue by wu-wenxiang** [openshift/openshift-ansible#11551]
#### Description

After apply patch : https://github.com/openshift/openshift-ansible/commit/9b2b62f1182320d00ce10aef5eaae09fc03c4454, I met this "undefined variable" issue in openshift_logging role.

> If role back to the previous version, everything OK.

##### Version

```
$ git describe
openshift-ansible-3.11.107-1-5-g9b2b62f

$ ansible --version
ansible 2.7.8
  config file = /home/cloud-user/openshift-ansible/ansible.cfg
  configured module search path = [u'/home/cloud-user/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Apr  9 2019, 14:30:50) [GCC 4.8.5 20150623 (Red Hat 4.8.5-36)]
```

##### Steps To Reproduce
1. git checkout 9b2b62f1182320d00ce10aef5eaae09fc03c4454
2. ansible-playbook -i demo2-inventory ~/openshift-ansible/playbooks/openshift-logging/config.yml

##### Expected Results
Describe what you expected to happen.

```
Should work as normal
```

##### Observed Results
Describe what is actually happening.

```
Logging Install : In Progress (0:00:01)
This phase can be restarted by running: playbooks/openshift-logging/config.yml

Failure summary:

Hosts: openshift2-ocp-app-1, openshift2-ocp-app-2, openshift2-ocp-app-3, openshift2-ocp-infra-1, openshift2-ocp-infra-2,
Play: Get common IP facts when necessary
Task: Gather Cluster facts
Message: The task includes an option with an undefined variable. The error was: 'vars_openshift_facts_system_facts' is unde

      The error appears to have been in '/home/cloud-user/openshift-ansible/playbooks/openshift-logging/private/config.y, but may
      be elsewhere in the file depending on the exact syntax problem.

      The offending line appears to be:

        tasks:
        - name: Gather Cluster facts
          ^ here
```

For long output or logs, consider using a [gist](https://gist.github.com/)

##### Additional Information

Provide any additional information which may help us diagnose the
issue.

* Your operating system and version, ie: RHEL 7.2, Fedora 23 (`$ cat /etc/redhat-release`)
* Your inventory file (especially any non-standard configuration parameters)
* Sample code, etc

```
CentOS 7.6

$ uname -a
Linux bastion 3.10.0-957.1.3.el7.x86_64 #1 SMP Thu Nov 29 14:49:43 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

$ cat tmp/demo2-inventory
[OSEv3:children]
masters
etcd
nodes

[OSEv3:vars]
ansible_ssh_user="cloud-user"
ansible_become=true

#cloudprovider
openshift_cloudprovider_kind=azure
openshift_cloudprovider_azure_cloud="AzureChinaCloud"
openshift_cloudprovider_azure_client_id="XXXXXXXX"
openshift_cloudprovider_azure_client_secret="XXXXXXXX"
openshift_cloudprovider_azure_tenant_id="XXXXXXXX"
openshift_cloudprovider_azure_subscription_id="XXXXXXXX"
openshift_cloudprovider_azure_resource_group="demo2"
openshift_cloudprovider_azure_location="chinaeast2"
#endcloudprovider

#openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true', 'challenge': 'true', 'kind': 'HTPasswdPasswordIdentityProvider'}]
openshift_disable_check=docker_image_availability

#oreg_auth_user=service_account 
#oreg_auth_password=service_account_token 
openshift_master_api_port=443
openshift_master_console_port=443
openshift_hosted_router_replicas=1
openshift_hosted_registry_replicas=1
openshift_master_cluster_method=native
openshift_master_cluster_hostname="demo2-master.test.com"
openshift_master_cluster_public_hostname="demo2-master.test.com"
openshift_master_default_subdomain="demo2-apps.test.com"
openshift_deployment_type=origin
openshift_examples_modify_imagestreams=true

container_runtime_docker_storage_setup_device=/dev/sdd
container_runtime_docker_storage_type=overlay2

# Storage Class change to use managed storage
openshift_storageclass_parameters={'kind': 'managed', 'storageaccounttype': 'Standard_LRS'}

# service catalog
openshift_enable_service_catalog=true
openshift_hosted_etcd_storage_kind=dynamic
openshift_hosted_etcd_storage_volume_name=etcd-vol
openshift_hosted_etcd_storage_access_modes=["ReadWriteOnce"]
openshift_hosted_etcd_storage_volume_size=SC_STORAGE
openshift_hosted_etcd_storage_labels={'storage': 'etcd'}

# metrics
openshift_metrics_install_metrics=true
openshift_metrics_cassandra_storage_type=dynamic
openshift_metrics_storage_volume_size=20Gi
openshift_metrics_hawkular_nodeselector={"node-role.kubernetes.io/infra": "true"}
openshift_metrics_cassandra_nodeselector={"node-role.kubernetes.io/infra": "true"}
openshift_metrics_heapster_nodeselector={"node-role.kubernetes.io/infra": "true"}

# logging
openshift_logging_install_logging=true
openshift_logging_es_pvc_dynamic=true
openshift_logging_storage_volume_size=50Gi
openshift_logging_kibana_nodeselector={"node-role.kubernetes.io/infra": "true"}
openshift_logging_curator_nodeselector={"node-role.kubernetes.io/infra": "true"}
openshift_logging_es_nodeselector={"node-role.kubernetes.io/infra": "true"}
openshift_logging_elasticsearch_memory_limit="8Gi"

# Setup azure blob registry storage
openshift_hosted_registry_storage_kind=object
openshift_hosted_registry_storage_azure_blob_accountkey="XXXXXXXXX"
openshift_hosted_registry_storage_provider=azure_blob
openshift_hosted_registry_storage_azure_blob_accountname="demo2hdd"
openshift_hosted_registry_storage_azure_blob_container=registry
openshift_hosted_registry_storage_azure_blob_realm=core.windows.net

[masters]
demo2-ocp-master-1

[etcd]
demo2-ocp-master-1

[nodes]
demo2-ocp-master-1 openshift_node_group_name="node-config-master"
demo2-ocp-infra-1 openshift_node_group_name="node-config-infra"
demo2-ocp-app-1 openshift_node_group_name="node-config-compute"
```

**Comment by wu-wenxiang** [comment]
@mtnbikenc https://github.com/openshift/openshift-ansible/commit/9b2b62f1182320d00ce10aef5eaae09fc03c4454

**Comment by wu-wenxiang** [comment]
Checked, this issue had been fix if update to openshift-ansible-3.11.110-1-12-g67cc596
I didn't check further that which patch fixed it.
Anyway, close :-)

**Comment by pavankjadda** [comment]
I faced similar issue this afternoon at 2 pm EST. Waiting for next update
